appserver/admingui/eventbus-notifier-console-plugin/src/main/resources/eventbus/eventbusNotifierConfiguration.jsf

```diff
@@ -105,11 +101,6 @@ holder.
                           helpText="$resource{i18nebn.notifier.eventbus.configuration.enabledLabelHelpText}">
                 <sun:checkbox id="enabledProp" selected="#{pageSession.enabledSelected}" selectedValue="true" />
             </sun:property>
-            <sun:property id="noisy" labelAlign="left" noWrap="#{true}" overlapLabel="#{false}" 
```

Comment:
I would also remove the string property from the `Strings.properties` file and the l10n equivalents.